### ISSUE TITLE
space between navbar and donation bar

### ISSUE THREAD

**Issue by shreyaa-s** [publiclab/PublicLab.Editor#446]
### Please describe the problem (or idea)
While testing the editor for issues related to the responsiveness of the editor I encountered additional space between the two bars.
 Screenshots depicting the same:
![Screenshot_20200402_134401](https://user-images.githubusercontent.com/55884943/78238257-00114800-74fa-11ea-93ba-4f835a315ec7.png)

![Screenshot_20200402_134454](https://user-images.githubusercontent.com/55884943/78238272-04d5fc00-74fa-11ea-9728-854141928113.png)

****

Thank you!

**Comment by NitinBhasneria** [comment]
The donation bar is actually disabled, but not reflected in the main site. So, this is not the issue. For clearing up please see publicLab/plots2. Thanks

**Comment by shreyaa-s** [comment]
@NitinBhasneria thanks for mentioning it.

**Comment by jywarren** [comment]
Yes, i'm going to move this issue over there and re-open it, as it is important! Thank you!!!
